flask_appbuilder/menu.py

```diff
@@ -123,3 +151,54 @@ def add_separator(self, category=""):
             raise Exception(
                 "Menu separator does not have correct category {}".format(category)
             )
+
+
+class MenuApi(BaseApi):
+    resource_name = "menu"
+
+    @expose('/', methods=["GET"])
```

Comment:
public endpoint, is this intentional? it really does not return any data when no user is authenticated

Reply:
Correct, it returns an empty array, currently. However if a user decides to expose a menu for the public role we'd still want to serve that [limited] menu, right?

Reply:
Right, yet we do have a public role for this. It's actually a decision between exposing an unauthenticated EP that anonymously return an empty list, or exposing an authenticated EP that returns 401